games/r.yaml

```diff
@@ -118,6 +118,29 @@
   type: remake
   updated: 2015-07-29
 
+- name: Rednukem
+  type: remake
+  originals:
+  - Redneck Rampage
+  - Duke Nukem 3D
+  repo: https://github.com/nukeykt/NBlood
+  url: https://nukeykt.retrohost.net/
+  development: very active
+  status: playable
+  lang:
+  - C++
+  framework:
+  - SDL
+  - SDL2
+  - OpenGL
+  license:
+  - GPL2
```

Comment:
Change `license` to `Custom`. Every Build engine game's license is custom due to Ken Silverman's buildlic.txt.